recipes-sota/libaktualizr-demo/libaktualizr-demo_git.bb

```diff
@@ -36,7 +37,10 @@ do_install_append () {
 
 FILES_${PN} += " \
   ${bindir}/libaktualizr-demo-app \
+  ${libdir}/libaktualizr.so \
+  ${libdir}/libaktualizr_secondary.so \
   ${libdir}/sota/sota.toml \
   ${libdir}/sota/conf.d \
   ${localstatedir}/sota/sota_provisioning_credentials.zip \
   "
+FILES_${PN}-dev = ""
```

Comment:
Why is this necessary?

Reply:
Otherwise, all shared libraries get into the PN-dev package and it leads to a QA error "PN rdepends on PN-dev" or something like that. We have it in meta-updater as well.